jcl/src/java.management/share/classes/module-info.java.extra

```diff
@@ -22,10 +22,9 @@
  *******************************************************************************/
 module java.management {
 	requires java.logging;
-	exports com.ibm.java.lang.management.internal to com.ibm.management, jdk.management;
-	exports sun.management.spi to com.ibm.management;
+	exports com.ibm.java.lang.management.internal to jdk.management;
 	/*[IF Open-Module-Support]*/
-	opens java.lang.management to com.ibm.management;
+	opens java.lang.management to jdk.management;
```

Comment:
This isn't something OpenJ9 should be adding. Please track down what needs this so we can determine what needs to be done to avoid it.

Reply:
Reflection is used in `ExtendedThreadInfoImpl.java` to access the `nativeTId` field in ThreadInfo object from the `java.management` module for use.

Reply:
We can create an internal access class to query this, and avoid the use of reflection and having to open the module.

Reply:
See java.lang.VMAccess for an example